setup.py

```diff
@@ -181,7 +181,7 @@ def package_files(prefix: str, directory: str, extension: str):
 ]
 
 extra_deps['streaming'] = [
-    'mosaicml-streaming<=0.1.*',
+    'mosaicml-streaming<0.2.*',
```

Comment:
Instead of 0.2.*, can we do <0.3? or should it be 0.2

Reply:
We need a star for pre-release version. We will change it soon once streaming goes out of pre-release.